cypress/support/commands/analytics.js

```diff
@@ -1,11 +1,9 @@
 // Should be moved into integration/pages/index.js once all pages have ATI
 Cypress.Commands.add('hasNoscriptImgAtiUrl', atiUrl => {
-  cy.get('noscript')
-    .eq(0)
-    .should(
-      'contain',
-      `<img height="1px" width="1px" alt="" style="position:absolute" src="${atiUrl}`,
-    );
+  cy.get('noscript').should(
```

Comment:
Yep, this will check each of the noscript tags for their content and return true if it matches the string. Not the most efficient test but I opted for it because we don't have comscore analytics to all pages so the position rendered for noscript tags vary depending on if comscore is present or not. 

For example, if comscore is rendered then the ATI noscript tag will be at index 1 but if the comscore noscript is not rendered, the ATI noscript will be at index 0